src/test/scala/mesosphere/marathon/tasks/InstanceTrackerImplTest.scala

```diff
@@ -60,40 +62,40 @@ class InstanceTrackerImplTest extends AkkaUnitTest {
     }
 
     def testList(call: InstanceTracker => InstanceTracker.InstancesBySpec)(implicit instanceTracker: InstanceTracker): Unit = {
-      val instance1 = makeSampleInstance(TEST_APP_NAME / "a")
-      val instance2 = makeSampleInstance(TEST_APP_NAME / "b")
-      val instance3 = makeSampleInstance(TEST_APP_NAME / "b")
+      val instance1 = Instance.Scheduled(AppDefinition(testAppId / "a"))
+      val instance2 = Instance.Scheduled(AppDefinition(testAppId / "b"))
+      val instance3 = Instance.Scheduled(AppDefinition(testAppId / "b"))
```

Comment:
This is so much nicer. Ideally we use `Instance.Provisioned(...)` and `Instance.Running(...)` factories instead of the test instance builders.